src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -133,17 +269,10 @@ def build_process_costs(
         process_contingency_percentage=15,
         # arguments related to Fixed OM costs
         nameplate_capacity=500,
-        labor_types=[
-            "skilled",
-            "unskilled",
-            "supervisor",
-            "maintenance",
-            "technician",
-            "engineer",
-        ],
-        labor_rate=[27.90, 23.26, 30.29, 24.06, 23.43, 46.82],
+        labor_types=None,
```

Comment:
Do you actually need all of these argument; they all look like things I just saw in the ConfigDict. If so, this seems like duplication of effect and an opportunity for something to go wrong. How do these arguments get used and populated?

Also, you should be able to remove the `# pylint: disable-next=dangerous-default-value` line above this.

Reply:
My point is that these already exist in the ConfigDict do they not? If that is the case, why create a new local variable for something that is already stored elsewhere.